src/cpdb/helpers.py

```diff
@@ -23,35 +23,46 @@
 where would this list comes from? """
 
 
-def get_data(branch, table) -> dict:
-    rv = {}
+def unpack_object(obj):
+    s = str(obj["Body"].read(), "utf8")
+    data = StringIO(s)
+    df = pd.read_csv(data, encoding="utf8")
+    return df
 
+
+def fetch_boto3_data(branch: str, table: str, previous=False):
+    if previous:
```

Comment:
would suggest maybe making this function dumber, and just passing in the key instead of a branch, table and previous flag. Makes it more extensible for pulling other types of objects

Reply:
would agree except then we have a "key constructor" function that is also pretty specific to cpdb and if we are not borrowing functions between different data products it is okay to leave like this for now?

Reply:
yeah I don't think its a blocker or anything